js/HelpTextComponent.js

```diff
@@ -68,11 +68,11 @@ HelpTextComponent.prototype.updateHelpSidebar = function () {
     var component = this,
         $activeTabContainer = component.$html.find('.tab__pane.is-active .tab__container'),
         activeTabSideBarContentHtml = component.$html.find('.tab__pane.is-active .tab__sidebar').html(),
-        $mobileToggleHelpButton = $('<a href="#" class="show-page-help js-show-page-help"><i class="icon-question-sign"></i><span class="hide">Show on page help</span></a>'),
+        $mobileToggleHelpButton = $('<button class="show-page-help js-show-page-help"><i class="icon-question-sign"></i><span class="hide">Show on page help</span></button>'),
         $pageMainTitle = component.$html.find('.main-title'),
         $tabHelp = component.$html.find('.tab-help'),
         isMobile,
-        $mobileCloseHelpButton = $('<a href="#" class="close-page-help js-close-page-help"><i class="icon-remove-sign"></i></a>');
+        $mobileCloseHelpButton = $('<button class="close-page-help js-close-page-help"><i class="icon-remove-sign"></i></button>');
```

Comment:
We should add `aria-hidden="true"` to this icon so it's not read by SRs and add either an aria-label or hidden span to the button for SRs.

Reply:
Done.